### ISSUE TITLE
Default values for LockTable parameters

### ISSUE THREAD

**Issue by jwikman** [MicrosoftDocs/dynamics365smb-devitpro-pb#2146]
What's the defaults for the parameters in `Record.LockTable(Wait, VersionCheck)`?

Also consider to sync the corresponding docs for RecordRef (https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/recordref/recordref-locktable-method). 

Thanks
Johannes

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f60835db-7fec-a288-a764-b84e604f3b64
* Version Independent ID: 6c3794bf-27b0-643c-b370-9ba9036da260
* Content: [Record.LockTable(\[Boolean\] \[, Boolean\]) Method - Business Central](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/record/record-locktable-method)
* Content Source: [dev-itpro/developer/methods-auto/record/record-locktable-method.md](https://github.com/MicrosoftDocs/dynamics365smb-devitpro-pb/blob/main/dev-itpro/developer/methods-auto/record/record-locktable-method.md)
* Service: **dynamics365-business-central**
* GitHub Login: @SusanneWindfeldPedersen
* Microsoft Alias: **solsen**

**Comment by dnpb** [comment]
`VersionCheck` is documented, although not too clearly:

> If this parameter is false, blank, **or not used**, the version will not be checked.

"not used" I presume means not passed, hence applying the default, which it states is `false`.

I agree that it can/should be clearer.

There are a lot of these issues and it makes me think MS need a way to document - and enforce documentation of - default parameter values.

**Comment by jwikman** [comment]
> There are a lot of these issues and it makes me think MS need a way to document - and enforce documentation of - default parameter values.

Agreed. I've created a few issues for default values when I see them are missing. But it seems to be a general issue, that I hope that they will have a more automated solution to. :-)
